GAD 6 - Verbessertes Sortieren/tests/simplesort/PivotFinderTester.java

```diff
@@ -508,8 +511,8 @@ public void MedianPivotFrontTest() {
         int[] array;
 
         array = new int[] {69, 420};
-        assertEquals(0, PivotFinder.getMedianPivotFront(3).findPivot(array, 0, 1));
-        assertEquals(0, PivotFinder.getMedianPivotFront(5).findPivot(array, 0, 1));
+        assertEquals(0, 1, PivotFinder.getMedianPivotFront(3).findPivot(array, 0, 1));
```

Comment:
>Sollte es keine eindeutige Mitte geben, soll die "linke Mitte" gewählt werden.

According to the change to this test be violating this rule? I mean, granted you have 100%, but I don't get the logic here

Reply:
I guess this only applies to "MidPivot" (where this requirement is from), not to the Median Pivots.

Reply:
Hmm okay, that may be true as well, since you have 100% on Artemis, I have no reason to doubt
Thanks for contributing